pywinauto/base_application.py

```diff
@@ -576,64 +577,117 @@ def _ctrl_identifiers(self):
 
         return control_name_map
 
-    def print_control_identifiers(self, depth=None, filename=None):
+    def dump_tree(self, depth=10, max_width=10, filename=None):
         """
-        Prints the 'identifiers'
+        Dump the 'identifiers' to console or a file
 
-        Prints identifiers for the control and for its descendants to
+        Dump identifiers for the control and for its descendants to
         a depth of **depth** (the whole subtree if **None**).
 
-        .. note:: The identifiers printed by this method have been made
+        :param depth: Max depth level of an element tree to dump (None: infinite).
+
+        :param max_width: Max number of children of each element to dump (None: infinite)
```

Comment:
`unlimited` sounds better than `infinite` in this context.

Reply:
Yes, it looks better actually.